boards/samr21-xpro/include/board.h

```diff
@@ -52,6 +52,21 @@ extern "C" {
                                      .reset_pin = GPIO_PIN(PB, 15)}
 
 /**
+ * @name    SX127x configuration
+ *
+ * {spi bus, nss pin, reset pin & dioX pins}
+ */
+
+#define SX127X_PARAMS_BOARD       { .spi       = SPI_DEV(1), \
```

Comment:
This is not part of the board but can be plugged to it from outside: the right place is in a separate `sx127x_params.h` that can be in `boards/samr21-xpro/include` directory. See `boards/arduino-due` for an example.

This comment is also valid with #7340.